sdk/storage/storage-internal-avro/package.json

```diff
@@ -68,7 +68,7 @@
     "puppeteer": "^2.0.0",
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
-    "@rollup/plugin-commonjs": "^11.0.1",
+    "@rollup/plugin-commonjs": "11.0.2",
```

Comment:
Why removing the `^`?

Reply:
No idea. `rush update` complains about this so updated it to the same one as other packages.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/9066

@rollup/plugin-commonjs update was causing build failures, so we agreed to pin the version to 11.0.2.

Reply:
I see